### PR TITLE
Alert & Buttons Component Implemented

### PR DESCRIPTION
**Pull request by anupkrbid**

### CODE REVIEW

File: docs/alerts.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </div>
+          </div>
+        </div>
+      </main>
+    </section>
+
+    <custom-footer></custom-footer>
```
Comment (Mohasin-Haque) [inline docs/alerts.html]: You can remove this space for sanity

**Review by Mohasin-Haque — COMMENTED** [review]
Everything seems good.. Please go through the listed comments once. You are good to go!!

File: docs/alerts.html (hunk 1 above)
Comment (tushargoyal812) [reply docs/alerts.html]: yes you can remove this space

**Review by tushargoyal812 — COMMENTED** [review]
take care of only one suggestion, good to merge.

File: docs/alerts.html (hunk 1 above)
Comment (anupkrbid) [reply docs/alerts.html]: ok

**Comment by anupkrbid** [comment]
Alert & Buttons Component Implemented

**Pull request merged**